src/main/java/com/example/books/exception/CustomGlobalExceptionHandler.java

```diff
@@ -34,6 +34,20 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
+    protected ResponseEntity<Object> handleRegistrationException(

```

Comment:
Here examples:
https://www.baeldung.com/exception-handling-for-rest-with-spring
```suggestion
    ExceptionHandler({ RegistrationException.class })
    public ResponseEntity<Object> handleRegistrationException(
```